### PR TITLE
Add Image Segmentation (Part 1) article

### PR DESCRIPTION
**Pull request by danielavornic**
Added the necessary files for the article focusing on the traditional methods of image segmentation.
There is a Jupyter notebook with the code used in the article, as well as a folder for the images included in the markdown file.

### CODE REVIEW

**Comment by pyramixofficial** [comment]
Could you add a `requirements.txt` file instead of having all the libraries in one line for the installation part?
And also make sure to indicate the python version that you were using in the requirements of the project
Lastly don't forget to modify the README once you make the changes

File: article-image_segmentation_1/article.md [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+
+Each segmentation type possesses distinct traits and finds relevance in diverse applications. In the subsequent section, we will explore some common image segmentation techniques with examples, showcasing their respective strengths and weaknesses.
+
+## Image Segmentation Techniques <a id="techniques"></a>
+
+Over the years, the field of image segmentation has witnessed the development of various techniques, broadly classified into two main categories: traditional methods and deep learning approaches, as depicted in Figure 3. While both categories offer valuable insights, this article will focus exclusively on traditional techniques. The discussion of deep learning methods will follow in Part 2.
+
+![Image Segmentation Techniques](images/techniques.png)
+
+*Figure 3. Image Segmentation Techniques. (I coded this diagram using Mermaid; could it be designed by someone in the design team?)*
```
Comment (danielavornic) [inline article-image_segmentation_1/article.md:73]: Could anyone from the design team help to create such a diagram or shall I do it myself? The current one doesn't look good.

Comment (pyramixofficial) [reply article-image_segmentation_1/article.md:73]: Yes, that could be done with the help of someone from the design team, I will reach out to them and let them know about it, thx for mentioning it!

**Review by pyramixofficial — APPROVED** [review]
LGTM!

**Pull request merged**